main_test.go

```diff
@@ -574,7 +574,7 @@ func listArchivedFiles(t *testing.T, body []byte) []byte {
 	var listing bytes.Buffer
 
 	for _, f := range zipReader.File {
-		listing.WriteString(fmt.Sprintf("%d %s\n", f.UncompressedSize64, f.Name))
+		listing.WriteString(fmt.Sprintf("%d %s\n", f.UncompressedSize64, filepath.ToSlash(f.Name)))
```

Comment:
Please add a comment here to remember that we cannot trust the format of `f.Name` and then we need to use `ToSlash`.